app/core/interfaces/map/interface.js

```diff
@@ -176,7 +177,7 @@ define([
       // Include the Google JSAPI for using Maps
       require(['https://www.google.com/jsapi'], function () {
         // Load Maps API using provided key, and call initializeMap() when API is loaded
-        google.load('maps', '3', {other_params: 'sensor=false&libraries=places&key=' + that.getApiKey(), callback: function () {
```

Comment:
Reason for removing `sensor` Reference: https://developers.google.com/maps/documentation/javascript/error-messages#sensor-not-required